backend/app/controllers/spree/admin/stores_controller.rb

```diff
@@ -40,7 +40,7 @@ def create
 
         if @store.save
           flash[:success] = flash_message_for(@store, :successfully_created)
-          redirect_to spree.admin_stores_path
+          redirect_to @store.formatted_url + spree.admin_stores_path
```

Comment:
@MatthewKennedy we will need a test for this also, we can test it in the controller_spec just to see if the URL generated is proper

Reply:
@damianlegawiec This will also fail a test in the feature tests too, I was trying to get around it using the `Capybara.app_host` as you used when you wrote the tests for a previous PR, but I couldn't get it to work.

Just going to fire up ngrok and test if this will require a fresh login for the admin user after create, as it will be on a different domain.